### ISSUE TITLE
Setting api_key through initializer configuration does nothing

### ISSUE THREAD

**Issue by zhyhun-ivan** [onfido/onfido-ruby#19]
Hi, i've set key inside initializer as described in readme like this:
```
Onfido.configure do |config|
  config.api_key = 'my_key_is_here'
  config.logger = Rails.logger
end
```

But when i call the code below the api_key is empty for `Onfido::API` instance, because in this [file](api.rb#L4) there's no default value taken from initialization configuration:
```
onfido = Onfido::API.new
```
Can you please fix this, thanks.

Also it would be great to have a class methods like **Onfido.applicant** or **Onfido.check** etc, instead of creating a new instance of `Onfido::API` and then `Onfido::Applicant` each time you want to make a call.

Much appreciate you attention and help on this issues.

**Comment by zhyhun-ivan** [comment]
For the first part of the issue regarding api_key - i have found that it's set inside initializer in `Onfido::Resource`, but following readme it's not obvious, as when you assign `onfido = Onfido::API.new` you can see that instance_variable **@api_key** is set to nil, which is confusing.

**Comment by ciaran16** [comment]
Hi @zhyhun-ivan, these issues will hopefully be addressed in our upcoming major version release (v2.0.0).

Config will be per `Onfido::API` instance, but once you create an instance, it won't re-create the `Onfido::Applicant` instance each time.

```rb
onfido = Onfido::API.new(api_key: ENV['ONFIDO_API_KEY'], region: :eu)

onfido.applicant.create(...)

# The Onfido::Applicant instance created on the previous line is re-used.
onfido.applicant.create(...)
```

This release should go out in the next few days.

**Comment by ciaran16** [comment]
Closing due to the release of v2.0.0, feel free to re-open if you have any issues.
